bin/connector_config/Generate.hs

```diff
@@ -37,38 +37,66 @@ data HolyWeapon
 holyWeaponLong :: HolyWeapon -> String
 holyWeaponLong = (++ " Received") . show @HolyWeapon
 
+data FillerItem
+    = AngelicRobe
+    | EnergyRing
+    | SecretBook
+    | Speedwings
+    | GoddessIcon
+    | DragonShield
+    | Talisman
+    | BodyRing
+    | Boots
+    | KnightCrest
+    | HeroCrest
+    | OrionsBolt
+    | GuidingRing
+    | ElysianWhip
+    | OceanSeal
+    | MasterSeal
+    deriving (Show, Enum, Bounded, Typeable)
+
 data Chapter
     = Prologue
     | -- For now, we treat Eirika and Ephraim versions of the same map as the
       -- same location
       C Int
+    | T Int
+    | R Int
     | C5x
     | Endgame
     | Victory
+    | R10
     deriving (Show, Typeable)
 
 instance Bounded Chapter where
     minBound = Prologue
-    maxBound = Victory
+    maxBound = R10
 
 instance Enum Chapter where
     toEnum 0 = Prologue
     toEnum 6 = C5x
     toEnum 22 = Endgame
     toEnum 23 = Victory
+    toEnum 41 = R10
     toEnum i
         | i < 1 = error $ "invalid chapter index " ++ show i
         | i < 6 = C i
         | i < 22 = C (i - 1)
+        | i < 32 = T (i)
```

Comment:
This is not really how this function works.

The goal of this function is to map a number from 0-[however many chapters we have] to that chapter following some ordering. So 0 -> Prologue, 1 -> Chapter 1, etc. Right now, you're mapping the number 23 to "Tower 23", which is not a real location. What we actually want to do is line them up like

Prologue
C 1
...
C5x
C 6
...
T 1
T 2
...
R 1
...
R 10

and then do some logic to map the numbers properly.